docs/development/upgrading-chrome.md

```diff
@@ -5,27 +5,42 @@ on each Chromium upgrade in Electron.
 
 ## Update libchromiumcontent (a.k.a. libcc)
 
-- Update the `VERSION` file at the root of the `electron/libchromiumcontent`
-  repository.
+- Clone the repo:
+```
+git clone git@github.com:electron/libchromiumcontent.git
+cd libchromiumcontent
+```
+- Run bootstrap script to init and sync git submodules:
+```
+./script/bootstrap -v
+```
+- Update `VERSION` file to correspond to Chromium version.
 - Run `script/update`, it will probably fail applying patches.
 - Fix failing patches. `script/patch.py` might help.
   - Don't forget to fix patches in the `patches-mas/` folder.
-- Make sure compilation succeeds on at least one supported platform/arch.
-- Open a pull request on `electron/libchromiumcontent` with the changes.
+- Build libcc:
+```
+./script/build
+```
+- Create dist folders which will be used by electron:
+```
+./script/create-dist --no_zip
+cd dist/main
+../../tools/generate_filenames_gypi filenames.gypi src shared_library static_library
+cd -
+```
+- Open a pull request to `electron/libchromiumcontent` with the changes.
 - Fix compilation on the all supported platforms/arches.
 
-## Fix Electron code
+## Update Electron
 
 - Set `vendor/libchromiumcontent` revision to a version with the new Chromium.
   - It will be great if GH builds for this libcc version are already green
     and its archives are already available. Otherwise everyone would need
     to build libcc locally in order to try build a new Electron.
 - Set `CLANG_REVISION` in `script/update-clang.sh` to match the version
-  Chromium is using in `tools/clang/scripts/update.py`.
-  If you don't have a Chromium checkout you can find it there:
-  https://chromium.googlesource.com/chromium/src.git/+/{VERSION}/tools/clang/scripts/update.py
-  (Just change the `{VERSION}` in the URL to the Chromium version you need.)
-- Don't forget to (re)run `script/bootstrap.py`.
+  Chromium is using. You can find it in file `src/tools/clang/scripts/update.py` in updated `electron/libchromiumcontent` repo.
```

Comment:
We should then specify two branches in the docs: if you have libcc locally (including how to symlink it or something), and if you download it from somewhere (current docs don't say how to do that)